pkg/server/container_status.go

```diff
@@ -88,3 +93,38 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		LogPath:     meta.LogPath,
 	}
 }
+
+// toCRIContainerInfo converts internal container object information to CRI container status response info map.
+func toCRIContainerStatusResponseInfo(ctx context.Context, container containerstore.Container, verbose bool) map[string]string {
```

Comment:
s/toCRIContainerStatusResponseInfo/toCRIContainerInfo? I saw also use that in the comment

Reply:
yeah couldn't decide .. I'll pick one.

Reply:
done..